storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc

```diff
@@ -6,9 +6,7 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-Logical Volume Manager Storage uses the TopoLVM CSI driver to dynamically provision local storage on the {product-title} clusters.
-
-{lvms} creates thin-provisioned volumes using Logical Volume Manager and provides dynamic provisioning of block storage on a limited resources cluster.
+Logical Volume Manager Storage uses lvm2 through the TopoLVM CSI driver to dynamically provision local storage on a cluster with limited resources.
```

Comment:
```suggestion
Logical Volume Manager (LVM) Storage uses LVM2 through the TopoLVM CSI driver to dynamically provision local storage on a cluster with limited resources.
```
I think this is the first mention of "Logical Volume Manager", so I believe the acronym should be included in parenthesis after it is spelled out. See, the [IBM Style Guide](https://www.ibm.com/docs/en/ibm-style?topic=grammar-abbreviations).

Reply:
There is are attributes for the product names in the [common attributes file](https://github.com/openshift/openshift-docs/blob/main/_attributes/common-attributes.adoc#L208-L209). It is best practice to use these attribute names when referring to the product.

Reply:
So, +1 to Janelle's comment, but use {lvms-fist} for the product name.